### ISSUE TITLE
DatePicker: Calendar without month picker is stretched to maximum width in Firefox and Chrome

### ISSUE THREAD

**Issue by Fensterbank** [OfficeDev/office-ui-fabric-react#2158]
### Bug Report

- __Package version(s)__: current
- __Browser and OS versions__: Mozilla Firefox, Google Chrome

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Normal 

#### Describe the issue:

When using the DatePicker if `isMonthPickerVisible={false}` the calendar view will use the maximum available width.
This makes this control broken in these major browsers when the month picker should not be displayed.
Picker works as expected in Internet Explorer 11 and Microsoft Edge.

#### Actual behavior:

In Mozilla Firefox and Google Chrome the calendar is stretched.
![Stretched Calendar](https://user-images.githubusercontent.com/1684826/27951061-e64fd5e0-6303-11e7-8768-5d359ed13d3a.png)

#### Expected behavior:

The calendar should not stretch, like it's in IE11 and Edge.
![Expected Calendar](https://user-images.githubusercontent.com/1684826/27951087-ff769f68-6303-11e7-948a-ddf5efdbea27.png)

**Comment by micahgodbolt** [comment]
Can you put together a quick codepen to demonstrate this bug? It's certainly something we should address asap.

**Comment by aadhars** [comment]
@Fensterbank I don't see this happening on latest master bits. As Micah suggested it would be good if you could share some codepen.

**Comment by Fensterbank** [comment]
okay, I'll try to reproduce it tomorrow at work.

**Comment by Fensterbank** [comment]
I found out that somewhere in my CSS there was a `table { width: 100% }` which affects the table in the month view.

But now it seems to be a general webcompat issue!
The way this picker works, the table uses all space it needs. So if I set table width to 100%, it takes all space it can get. For me it seems Chrome and Firefox are doing it right (event if the control is broken now) and Edge and IE11 are doing it wrong.

Because codepen doesn't want to save anything, I made a jsFiddle: https://jsfiddle.net/j1fdpr30/1/

It *could* be adressed by a pull request which sets the table width to `auto` inside of a DatePicker, but I'm not sure, if we should do it.
I broke the table and so I broke the control but Edge and IE don't care. ;-)
